ra/registration-authority.go

```diff
@@ -233,7 +238,7 @@ func (ra *RegistrationAuthorityImpl) NewCertificate(req core.CertificateRequest,
 
 	// Create the certificate and log the result
 	var cert core.Certificate
-	if cert, err = ra.CA.IssueCertificate(*csr, regID); err != nil {
+	if cert, err = ra.CA.IssueCertificate(*csr, regID, earliestExpiry); err != nil {
 		logEvent.Error = err.Error()
 		return emptyCert, nil
```

Comment:
This appears to be causing the Travis failure at https://travis-ci.org/letsencrypt/boulder/builds/64517400